src/test/prediction_service_test.cpp

```diff
@@ -844,6 +844,30 @@ TEST_F(TestPredict, PerformInferenceWithBinaryInputChangeModelInputLayout) {
     checkOutputValues(response, {37.0, 37.0, 28.0, 28.0, 238.0, 238.0}, INCREMENT_1x3x4x5_MODEL_OUTPUT_NAME);
 }
 
+/**
+ * Scenario - perform inference with binary input with witdth exceeding shape range when model shape is dynamic. Check results.
+ *
+ * 1. Load model with dynamic shape and input layout=nhwc, initial internal layout: nchw
+ * 2. Do the inference with single binary image tensor with witdth exceeding shape range - expect status OK and reshaped output tensor
+ */
+TEST_F(TestPredict, PerformInferenceWithBinaryInputAndShapeDynamic) {
+    using namespace ovms;
+
+    // Prepare model with changed layout to nhwc (internal layout=nchw)
+    ModelConfig config = INCREMENT_1x3x4x5_MODEL_CONFIG;
+    config.setBatchingParams("auto");
```

Comment:
Why is it auto? We do not use this feature in this test. Model batch size is 1 and request batch size i 1 as well